src/Templates/translatable/widgets/participantRow.twig

```diff
@@ -11,6 +11,12 @@
 {% elseif person.getUser %}
     - {{ person.user.email }}
 {% endif %}
+{% if person.deals %}
+    - {% for participantDeal in person.deals %}
+    {{participantDeal.slug}}{%if not participantDeal.isDone %} ❌{% else %} ✅{% endif %}{{ not loop.last ? '; ' }}
```

Comment:
- forcyklus super a hezké (bez trailing za posledním elementem), jen bych tam přidal pipe `|` místo středníku jako oddělovač, co už máme všude jinde
 - přidat mezery mezi svorky a výraz, aby to bylo všude stejné
```suggestion
    {{ participantDeal.slug }}{% if not participantDeal.isDone %} ❌{% else %} ✅{% endif %}{{ not loop.last ? '; ' }}
```